nav2_velocity_smoother/src/velocity_smoother.cpp

```diff
@@ -333,15 +338,36 @@ void VelocitySmoother::smootherTimer()
   }
 
   // Apply absolute velocity restrictions to the command
-  command_->twist.linear.x = std::clamp(
+  if(!is_6dof_) {
+    command_->twist.linear.x = std::clamp(
     command_->twist.linear.x, min_velocities_[0],
```

Comment:
I think you have some formatting issues. The line extension after the `std::clamp` should have an intend. Ditto for this entire block

Reply:
After going through the full thing, it looks like from this point to the end there's alot of lines that should have been indented that weren't from the original code and your added code. Please fix that :-)